drivers/base/bootvid/i386/pc/vga.c

```diff
@@ -57,13 +57,13 @@ static ULONG lookup[16] =
 
 ULONG_PTR VgaRegisterBase = 0;
 ULONG_PTR VgaBase = 0;
-static BOOLEAN ClearRow = FALSE;
```

Comment:
And here:
https://git.reactos.org/?p=reactos.git;a=blob;f=drivers/base/bootvid/arm/bootvid.c;hb=592bafd60a33b82e62174f124ce80b2e0374a03a#l8